Model/Api/OrderManagement.php

```diff
@@ -166,6 +167,14 @@ public function manage(
                 $this->saveUpdateOrder($reference, $type, $display_id, $storeId);
             }
             $this->metricsClient->processMetric("webhooks.success", 1, "webhooks.latency", $startTime);
+        } catch (BoltException $e) {
+            $this->bugsnag->notifyException($e);
+            $this->metricsClient->processMetric("webhooks.failure", 1, "webhooks.latency", $startTime);
+            $this->response->setHttpResponseCode(422);
+            $this->response->setBody(json_encode([
+                'status' => 'failure',
```

Comment:
Status is 'error' if other exceptions are thrown below, not 'failure'. Please check what's correct.

Reply:
I am sure about "failure" for 'cart.create' because it works with bolt server, see screenshots above.

I don't know about the right error answer for webhooks, because bolt server accepts in the same way answers with "failure" and "error". Actually it seems like it doesn't parse answer.

Let's left "error" for webhooks right now. I've created asana to investigate it.
https://app.asana.com/0/941920570700290/1162655511300921